src/coreclr/vm/interpexec.cpp

```diff
@@ -1486,6 +1491,9 @@ void InterpExecMethod(InterpreterFrame *pInterpreterFrame, InterpMethodContextFr
                     ip += 5;
 
                     InvokeCalliStub(LOCAL_VAR(calliFunctionPointerVar, PCODE), pCallStub, stack + callArgsOffset, stack + returnOffset);
+#else
+                    PORTABILITY_ASSERT("Attempted to execute calli instruction on wasm, this is not yet implemented");
```

Comment:
Can we put these in the InvokeXxx functions instead since they get used in various places, instead of ifdefing it into the call sites?